ui/src/shared/components/AddResourceDropdown.tsx

```diff
@@ -11,11 +12,21 @@ import {
   ComponentStatus,
 } from '@influxdata/clockface'
 
+// Actions
+import {showOverlay, dismissOverlay} from 'src/overlays/actions/overlays'
+
+// Types
+import {LimitStatus} from 'src/cloud/actions/limits'
+
+// Constants
+import {CLOUD} from '../constants'
+
 interface OwnProps {
   onSelectNew: () => void
   onSelectImport: () => void
   onSelectTemplate?: () => void
   resourceName: string
+  limitStatus?: LimitStatus
```

Comment:
Is this prop really optional? couldn't see defaults anywhere.

Reply:
Ya, this needs to be optional because there are instances of this that are used without this prop. I added a default at the location it's used.